include/flatbuffers/util.h

```diff
@@ -353,6 +417,7 @@ inline int FromUTF8(const char **in) {
   return ucc;
 }
 
+#ifndef FLATBUFFERS_PREFER_PRINTF
```

Comment:
Please indent #preprocessor commands here and elsewhere. use // clang-format: off where needed

Reply:
Done in 2b41739

Reply:
Should it indent the whole block (as in this case) even if the #ifdef is at 0 indentation? I got all the other cases where it was #ifdef / #endif was clearly within braces.